### DISCUSSION TITLE
Help Galaxy Brain Badge

### DISCUSSION THREAD

**Discussion by Fldzz** [community/community#19293] [Other Feature Feedback, Questions, & Ideas] [answered]
Please help me and tell me how i can get this badge?

**Comment by IvailoPetrov06** [comment, marked as answer]
When you answer on two discussions after you wil recive Galaxy Brain for more details you can see here https://github.com/Schweinepriester/github-profile-achievements

**Reply by Fldzz** [reply]
Thanks you!

**Comment by GreatHorse115** [comment]
Thank you. I learned a lot.
